assets/js/outround.js

```diff
@@ -0,0 +1,168 @@
+import $ from "jquery";
```

Comment:
IMO we should try to share the code with the normal pairings file as much as possible. I can help with refactoring JS but I definitely want to reduce duplication here before this is finalized